src/settings/project/kv_namespace.rs

```diff
@@ -0,0 +1,28 @@
+use crate::settings::binding::Binding;
+use std::fmt;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct KvNamespace {
+    pub id: String,
+    pub binding: String,
+}
+
+impl fmt::Display for KvNamespace {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "id: {}, binding: {}", self.id, self.binding)
+    }
+}
+
+impl std::cmp::PartialEq for KvNamespace {
```

Comment:
i *believe* this is the same as if you derived it, is that true? if it's not, might be worth a comment explaining why this is impl'd manually

Reply:
mmm good point. i'll have a look.